### DISCUSSION TITLE
Version 2.0.0

### DISCUSSION THREAD

**Discussion by samerton** [NamelessMC/Nameless#2970] [Releases]
## Version 2.0.0
After almost 7 years in development, version 2.0.0 is officially here! Thank you to everybody who has supported Nameless in this time - without your help it simply wouldn't have been possible to reach this point.

There shouldn't be any breaking changes to frontend templates or modules in this update (assuming you're coming from v2 pre-release 13) so you should be able to upgrade straight away!

A good overview of the new features in v2 can be found in the [readme](https://github.com/NamelessMC/Nameless/blob/v2/README.md).

You can find a changelog for the release [here](https://github.com/NamelessMC/Nameless/blob/v2/CHANGELOG.md#200---2022-08-05).

## Which zip should I download?

If you are updating from 2.0.0 pre-release 13, please download version 2 through your StaffCP's Update tab.
If you are installing a new website, download nameless-deps-dist.zip or nameless-deps-dist.tar.xz which will include all the files you need for a fresh start.

<hr /><em>This discussion was created from the release <a href='https://github.com/NamelessMC/Nameless/releases/tag/v2.0.0'>Version 2.0.0</a>.</em>

**Comment by DominikSLK** [comment]
🎉

**Comment by PadowYT2** [comment]
:tada:

**Comment by agnihackers** [comment]
@samerton are you happy to assign a CVE? please confirm, then only admin can move further

![Issue](https://user-images.githubusercontent.com/58265761/184480168-4766ece7-33e8-4b58-ad6f-6d13576c371b.jpg)

Reports:

https://huntr.dev/bounties/df06b7d7-6077-43a5-bd81-3cc66f0d4d19/
https://huntr.dev/bounties/c216db15-fe2f-42a7-852a-6c47498cf069/
